backend/src/crd/kubernetes/Makefile

```diff
@@ -0,0 +1,105 @@
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.31.0
```

Comment:
I think we can remove all of the envtest setup in the Makefile since KFP's tests are on Kind clusters and not envtest unless you expect that to change.